src/appMain/sdl_preloaded_pt.json

```diff
@@ -3541,7 +3541,9 @@
                             "name": "pressureTelltale",
                             "key": "OEM_REF_PRES_TEL",
                             "type": "WarningLightStatus",
-                            "mandatory": true
+                            "mandatory": true,
+                            "since": "2.0",
+                            "untill": "7.1"
```

Comment:
Please correct type in `until` for all sub params.

Reply:
@atiwari9 Done in 3cee9a1